src/state/types.ts

```diff
@@ -189,7 +189,9 @@ export type WhiteboardSlideInstance = {
    * @deprecated to be replaced with getActiveElementIds
    * */
   getActiveElementId(): string | undefined;
-  /** Return the active elements */
+  /**
+   * @returns IDs of the currently selected elements in the order of their selection.
```

Comment:
We use active that has a context of selected in many places, so lets change a bit wording, but thanks for correcting the comment.
```suggestion
Return ids of the currently active elements in the order of their selection.
```